appinventor/appengine/src/com/google/appinventor/client/wizards/FileUploadWizard.java

```diff
@@ -103,11 +103,7 @@ public void execute() {
         String uploadFilename = upload.getFilename();
         if (!uploadFilename.isEmpty()) {
           final String filename = makeValidFilename(uploadFilename);
-          if(!TextValidators.isValidCharFilename(filename)){
-            createErrorDialog(MESSAGES.malformedFilenameTitle(), MESSAGES.malformedFilename(),
-              Error.NOFILESELECETED, folderNode, acceptableTypes, fileUploadedCallback);
-            return;
-          } else if (!TextValidators.isValidLengthFilename(filename)){
+          if (!TextValidators.isValidLengthFilename(filename)){
```

Comment:
I'm not sure that we want to fully remove this check. For example, project files are AIAs so filenames should still be verified against allowable characters in ZIP file entries, and since APKs/IPAs will be installed on Android/iOS we should make sure the filename characters are allowable on both of those systems. Otherwise, someone might be able to construct a project that they can't compile/open.

Reply:
I don't think, this will cause an problem cause we have different wizards for importing project, also i think FileUploadWizard is specifically made for uploading media assets.

Reply:
I don't think that will cause an problem cause FileUploadWizard is specifically made for only uploading media assets file